packages/jumpstarter/jumpstarter/client/core.py

```diff
@@ -128,3 +130,19 @@ async def resource_async(
                 yield ResourceMetadata(**rstream.extra(MetadataStreamAttributes.metadata)).resource.model_dump(
                     mode="json"
                 )
+
+    def __log(self, level: int, msg: str):
+        self.logger.log(level, msg)
+
+    @asynccontextmanager
+    async def log_stream_async(self):
+        async def log_stream():
+            async for response in self.LogStream(empty_pb2.Empty()):
+                self.__log(logging.getLevelName(response.severity), response.message)
```

Comment:
```suggestion
                self.__log(logging.getLevelName(response.severity), response.message)
```

Can we make this controllable (via flag switch on the CLI?) also, be able to set the verbosity filter on this side i.e. may be as a client you want to see all, or you want to filter starting on Error logs?

I'll leave it for follow up, merging now :)